app/serializers/api/v3/statement_serializer.rb

```diff
@@ -1,17 +1,25 @@
 module API
   module V3
-    class StatementSerializer < Blueprinter::Base
-      identifier :id
+    class AttributesSerializer < Blueprinter::Base
```

Comment:
Not that it matters now while we have just one serialiser, but if multiple serialisers have an `AttributesSerializer` within would we end up with clashes?